kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java

```diff
@@ -38,11 +40,6 @@
     @XStreamAlias("score")
     private ScoreWrapper scoreWrapper;
 
-    @XmlElement(name = "planning-problem")
-    @XStreamAlias("planning-problem")
-    @XmlJavaTypeAdapter(JaxbUnknownAdapter.class)
-    private Object planningProblem;
-
```

Comment:
why delete planning-problem?
I use [POST] to execute optaplanner, but there occurs 500 Internal Server Error
they say there is no instance SolverInstance.planning-problem.